rules/explicit-length-check.js

```diff
@@ -154,6 +154,12 @@ function create(context) {
 			let node;
 			let autoFix = true;
 
+			const staticValue = getStaticValue(lengthNode, context.getScope());
+			if (staticValue && typeof staticValue.value !== 'number') {
+				// Ignore known, non-number length properties.
```

Comment:
Let's get this even stricter, only allow positive integers.

Reply:
Good point! Fixed.

Reply:
Actually, I mean use `Number.isInteger()` exclude `1.2` too.

Reply:
Fixed.